go.mod

```diff
@@ -5,6 +5,7 @@ go 1.16
 require (
 	github.com/AlekSi/gocov-xml v1.0.0
 	github.com/BurntSushi/toml v0.3.1
+	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751
```

Comment:
why?

Reply:
Hmm, it was required by generated `docs/swagger/docs.go`, but that file was not removed in `Make clean` 
So if I ran `go mod tidy` after a `make build`, this will be added automatically.